Update Java SDK docs to 3.0 (#2239)
getsentry/sentry-docs: src/platforms/java/common/legacy/log4j/index.mdx

---
title: log4j 1.x
sidebar_order: 30
---

<Alert level="warning" title="Note"><markdown>

> A new Java SDK has superseded this deprecated version. Sentry preserves this documentation for customers using the old client. We recommend using the [updated Java SDK](/platforms/java/) for new projects.

Reviewer: ```suggestion
An [updated Java SDK](/platforms/java/) supersedes this deprecated version. Sentry preserves this documentation for customers using the old client. We recommend using the updated Java SDK for new projects.
```
